app/src/main/java/org/eyeseetea/malariacare/layout/adapters/survey/DynamicTabAdapter.java

```diff
@@ -242,6 +243,34 @@ public void onClick(View v) {
         //Show confirm on full screen
         rootView .findViewById(R.id.options_table).setVisibility(View.GONE);
         rootView .findViewById(R.id.confirm_table).setVisibility(View.VISIBLE);
+
+        //Show question image in counter alert
+        ImageView imageView=(ImageView) rootView.findViewById(R.id.questionImageRow);
```

Comment:
@idelcano can this line go inside the if so we don't have to execute it every time? After our last effort improving the performance, I am little bit worried about this kind of stuff